backend/app/utils/Neo4jHelper.scala

```diff
@@ -15,6 +15,7 @@ import scala.collection.mutable
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.control.NonFatal
 import scala.collection.JavaConverters._
+import utils.Logging
```

Comment:
don't understand how this one even worked before! because they're both in a package called `utils`?

Reply:
hmm I got confused by this one too, turns out `import org.neo4j.driver.v1._` from above brought in `org.neo4j.driver.v1.Logging` which in 2.13 seems to take precedence over the one in the `utils` package. By importing `utils.Logging` I was then shadowing with the correct import again.

Digging in a bit more it looks like the way it worked before wasn't really supposed to work at all by the spec so they fixed it in 2.13: https://github.com/scala/scala/pull/6589. Nic hit the same thing in Riff-Raff: https://github.com/guardian/riff-raff/pull/666